drizzlepac/haputils/svm_quality_analysis.py

```diff
@@ -937,15 +936,19 @@ def report_wcs(total_product_list, json_timestamp=None, json_time_since_epoch=No
 
                 icnt = 0
                 # Restore an alternate to be the primary WCS
+                # Examples of WCS name formats:
+                # Default WCS name: IDC_xxxxxxxxx
+                # A priori WCS name: IDC_xxxxxxxxx-HSC30
+                # A posteriori WCS name: IDC_xxxxxxxxx-FIT_REL_GAIADR1
                 for key, value in dict_of_wcskeys_names.items():
                     if key != ' ':
                         alt_key = key
                         alt_wcs_name = value.strip()
 
-                        if len(alt_wcs_name) == 13 and alt_wcs_name.startswith('IDC_'):
+                        if '-' not in alt_wcs_name and alt_wcs_name.startswith('IDC_'):
                             alt_name = 'AlternateWCS_default'
                             delta_name = 'DeltaWCS_default'
-                        elif list(filter(alt_wcs_name.endswith, apriori_suffix_list)) != []:
+                        elif '-' in alt_wcs_name and 'FIT' not in alt_wcs_name:
```

Comment:
This logic would appear, to me at least, to allow 'IDC_12345678q-SVM_REL_GAIADR2' to be identified as an apriori solution.  Unfortunately, if there is an '-' in the WCSNAME, any of the aposteriori_list values could be present, not just 'FIT'.

Reply:
I tried to look for a listing of all the possible "a_posteriori" names in order to setup the logic properly for the "a priori" discriminant, and I found this

<Starting WCS>-FIT_<REL|IMG>_<Astrometric Catalog>

which is why I used the logic you see.  Looking at some of the SVM WCSNAMES and poking in the source code, it seems the "a posteriori" WCS is

<Starting WCS>-FIT_<REL|IMG|SVM>_<Astrometric Catalog>

I guess I did not piece together the definitive list.  Obi-Wan, where is a definitive list?

Reply:
The definitive list should be in the Drizzlepac readthedocs, something which has not been updated with all the latest options.  This part of the WCSNAME gets defined during SVM/MVM processing using the `align_to_gaia()` method of the `Product` class through the use of the fit_label parameter which gets passed to the `AlignmentTable.apply_fit()` method.   Now that we have this implemented, your experience confirms that we really need to update our documentation, including the docstrings for these methods.

Reply:
No worries.